cmd/xl-v1_test.go

```diff
@@ -37,7 +37,7 @@ func TestStorageInfo(t *testing.T) {
 	}
 
 	// Get storage info first attempt.
-	disks16Info := objLayer.StorageInfo()
+	disks16Info := objLayer.StorageInfo(nil)
```

Comment:
I think we can expect someone to provide context.Context always such as we can have a default value.  We can have a global

```
 var NoContext = context.background()
```

The reason is it looks pretty odd to pass nil for first argument to a function.

Reply:
Having a default global NoContext is not correct as the "callee" functions can add to the context in future and it will end up affecting the global context.

we can pass a new `context.Background()` instead of nil.